workflows/scripts/convert.py

```diff
@@ -0,0 +1,61 @@
+"""
+script: covert.py
+
+converts sqlite (or other formats) into parquet files
+"""
+import pathlib
+from typing import Union, List
+
+import cytotable
+
+
+def sqlite_to_parquet(
+    input_file: Union[str, List[str]],
+    out_path: str,
+    target_ext: str,
+    convert_configs: str,
+):
+    # checking if user has parquet file
+    if target_ext == ".parquet" and pathlib.Path(input_file).suffix != ".sqlite":
+        raise ValueError(
```

Comment:
it doesn't always. I think CytoTable can handle CSVs distributed across multiple folders. Please correct me if i'm wrong @d33bs !

Reply:
@axiomcura , @gwaybio - Yes, this is correct. CytoTable will attempt to navigate a provided source data directory for nested CSV's or SQLite data and concatenate or join the data together in the results.

Reply:
Sound good! created issue for future implementation #43